README.md

```diff
@@ -1,28 +1,38 @@
+# Comparing single molecule minimizations with different all atom force fields
 
-    ==============
-    |  CONTENTS  |
-    ==============
-    
-Last updated: Feb 27 2017
+**NOTE: only Python 2 is supported for the time being.**
 
-NOTE: only Python 2 is supported for the time being.
+### Project Goals
 
+This project relates to the larger goals of the 
+[Open Force Field Group's](https://github.com/open-forcefield-group)
+effort to automate force field parameterization. 
+
+Here we have created scripts to minize small molecules using a variety of force fields with two goals in mind. 
+1. Find places where the SMIRNOFF99Frosst force field behaves differently from currently accepted force fields.
+2. Discover molecules that are minized to different conformations by different force fields. These molecules will likely be used in future SMIRNOFF parameterizations. 
+
+### Force fields 
+
+The follow is a list of the force fields being considered here:
+
+* [SMIRNOFF99Frosst](https://github.com/open-forcefield-group/smirff99Frosst)
+* GAFF
+* GAFF2
+* MMFF94
+* MMFF94S
+* OPLS3
+* OPLS2005
 
 ### Contents
 
 * *ffcompare.py*: Script to read in mol2 files and run minimization using OpenMM (or oechem.OESzybki).
 
 * *genTriposGAFFandGAFF2.py*: Script to generate mol2 files with Tripos, GAFF, and GAFF2 atom types.
 
-TODO: remove anything related to 'commands' module.
```

Comment:
Did we move this `TODO` somewhere else? We don't want to delete / forget it I presume...